### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mathieuroques**

### CODE REVIEW

**Comment by mathieuroques** [comment]
- [] Create diagram for first page
- [] Replace screenshot by gif
- [] Clean the `hello word` script

File: README.md [hunk 1]
```diff
@@ -31,165 +31,202 @@
...
-
-# You can run the test suite locally to verify that it reproduces the issues
+```
+You can run the test suite locally to verify that it reproduces the issues
+```python
 test_suite.run()
 ```
+# ❓ FAQ
+<details>
+  <summary>About the ML worker</summary>
```
Comment (luca-martial) [inline README.md:133]: "What is a ML worker?" -> should be a question

File: README.md [hunk 2]
```diff
@@ -31,165 +31,202 @@
...
 ```python
 test_suite = scan_results.generate_test_suite("My first test suite")
-
-# You can run the test suite locally to verify that it reproduces the issues
+```
+You can run the test suite locally to verify that it reproduces the issues
+```python
 test_suite.run()
 ```
+# ❓ FAQ
```
Comment (luca-martial) [inline README.md:131]: You can add a "Where can I get more help?" or something similar and link the discord

File: README.md [hunk 3]
```diff
@@ -31,165 +31,202 @@
...
+    test_suite.upload(client, "my_project")
+    
+    ```
+
 
-## How to contribute
+# 👋 Community
 We welcome contributions from the Machine Learning community!
 
 Read this [guide](CONTRIBUTING.md) to get started.
```
Comment (luca-martial) [inline README.md:222]: should be on the same line as sentence above

File: README.md [hunk 4]
```diff
@@ -31,165 +31,202 @@
...
 
-<br />
-
-## Like what we're doing?
-
 🌟 [Leave us a star](https://github.com/Giskard-AI/giskard), it helps the project to get discovered by others and keeps us motivated to build awesome open-source tools! 🌟
```
Comment (luca-martial) [inline README.md:224]: before the leave us a star section there should be a "Join our thriving community ... " with a link to our discord

**Review by luca-martial — CHANGES_REQUESTED** [review]

**Comment by mathieuroques** [comment]
> TODO @luca-martial :
> 
> * [x]  Record higher quality gif
> * [x]  Write better Open-source and Closed-source description
> * [ ]  Decide if FAQ should be technical or not technical
> * [ ]  Add more references to our documentation
> * [x]  Write introduction paragraph about gsk
> * [x]  Add python version !!
> * [ ]  Create diagram for first page

**Review by luca-martial — APPROVED** [review]

**Comment by luca-martial** [comment]
@andreybavt this looks good to go on my end. We can make more changes on wording once Hub goes closed-source

**Pull request merged**